examples/hotrod/services/driver/redis.go

```diff
@@ -54,6 +54,7 @@ func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
 		ext.SpanKindRPCClient.Set(span)
 		defer span.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span)
+		r.logger.For(ctx).Info("FindDriverIDs at " + location)
```

Comment:
that's not useful. I would rather have this just before the `return` statement and log the `drivers` array.

Reply:
Got it OK